.github/scripts/nm-run-lm-eval-gsm-hf-baseline.sh

```diff
@@ -14,23 +14,19 @@ usage() {
     echo
     echo "  -m    - huggingface stub or local directory of the model"
     echo "  -b    - batch size to run the evaluation at"
-    echo "  -d    - device to use (e.g. cuda, cuda:0, auto, cpu)"
```

Comment:
turns out this doesn't work, need to pass `parallelize=True` to model args to use accelerate 🤦

Reply:
nice find